pkg/cmd/bundle/push.go

```diff
@@ -109,6 +113,15 @@ func (p *pushOptions) parseArgsAndFlags(args []string) error {
 		p.bundleContents = append(p.bundleContents, string(contents))
 	}
 
+	p.annotations = map[string]string{}
+	for _, annParam := range p.annotationParams {
+		if k, v, ok := strings.Cut(annParam, "="); ok {
+			p.annotations[strings.TrimSpace(k)] = strings.TrimSpace(v)
+		} else {
```

Comment:
can we make use of [ParseParams](https://github.com/tektoncd/cli/blob/main/pkg/params/mergeparams.go#L117) function here?

Reply:
Refined `ParseParams` as well